packages/react-percy-config/src/__tests__/defaults-tests.js

```diff
@@ -1,35 +1,177 @@
 import defaults from '../defaults';
+import micromatch from 'micromatch';
 
-describe('snapshotRegex', () => {
+const expectFileToMatchPatterns = (file, patterns) => {
+  const matches = micromatch([file], patterns);
+  expect(matches).toEqual([file]);
+};
+
+const expectFileNotToMatchPatterns = (file, patterns) => {
+  const matches = micromatch([file], patterns);
+  expect(matches).toEqual([]);
+};
+
+describe('snapshotIgnorePatterns', () => {
```

Comment:
This feels like a case where one-line test cases definitely adds a bunch of overhead and makes things
 harder to read — could be nice to simplify this down to one test case with an array of the 21+ paths that should be correctly ignored and just iterate through them. Alternatively, you could "generate the tests cases" like here: https://github.com/percy/percy-web/blob/master/tests/integration/components/build-info-dropdown-test.js#L20